### ISSUE TITLE
add max_combinations to the schema combinations settings

### ISSUE THREAD

**Issue by marina-p** [microsoft/restler-fuzzer#284]
The user should be able to limit the number of header or query combinations separately from the overall limit of max_combinations. 
 
The test combinations settings should be modified as follows:

```
  "test_combinations_settings": {
     "header_param_combinations" : {
          "param_type": "all",
          "max_combinations": 100
     },
     "example_payloads" : {  <-- all is assumed
          "max_combinations": 100
     },
}
```

**Comment by PatGod** [comment]
@marina-p Isn't this issue addressed by PR #320 ? Can we close this?

**Comment by marina-p** [comment]
Closed in PR #320
